### ISSUE TITLE
Take amounts from rule, not bill itself

### ISSUE THREAD

**Issue by JC5** [firefly-iii/firefly-iii#1332]
For the next version. This makes a bill more of a placeholder and not really a container for rules.

**Comment by gavu** [comment]
Hi, just a question regarding amounts.

I have set up my rules the following way: one rule assignes the transaction, renames it, **and** links it to the bill, so, I actually disabled your "migrated" bill rules, as I did not want to manage 2 different sets for every bill.

I noticed in your migrated rules, you automatically added the amounts, which, I presume, would also automatically update once you changed them in the bill itself (or maybe not?). 

Since I moved away from the migrated rules, I do not have min/max amounts in my rules, but only in the bill description. Will that affect my rules in any way?
